bitset.go

```diff
@@ -205,6 +205,28 @@ func (b *BitSet) Test(i uint) bool {
 	return b.set[i>>log2WordSize]&(1<<wordsIndex(i)) != 0
 }
 
+// Word retrieves bits i through i+63 as a single uint64 value
+func (b *BitSet) Word(i uint) uint64 {
```

Comment:
Is it intuitive to you that Word(i) would retrieve an uint64 starting at bit index i?

What about something more explicit:

- GetWord64AtBit
- ExtractUInt64At
...

Reply:
GetWord64AtBit sounds great, I'll change the name